frontend/src/container/WorkspaceContent.jsx

```diff
@@ -260,7 +260,7 @@ export class WorkspaceContent extends React.Component {
       type: BREADCRUMBS_TYPE.CORE,
       label: workspaceLabel
     }, {
-      link: <Link to={PAGE.WORKSPACE.CONTENT_LIST(state.workspaceInInUrl)}>{props.t('Contents')}</Link>,
+      link: <Link to={PAGE.WORKSPACE.CONTENT_LIST(state.workspaceIdInUrl)}>{props.t('Contents')}</Link>,
```

Comment:
not that this will break with the refactor of breadcrumbs from https://github.com/tracim/tracim/pull/4046

to update, if the other PR is merged first:
``` jsx
link: PAGE.WORKSPACE.CONTENT_LIST(state.workspaceIdInUrl)},
```
and add property `isALink: true`

Reply:
Thanks for the head's up. I'll leave this conversation open.